VSRAD.Package/Utils/BreakModeConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace VSRAD.Package.Utils
+{
+    static class BreakModeConverter
+    {
+        static public readonly string[] BreakModeOptions = new[] {  "Single active breakpoint, round-robin",
+                                                                    "Single active breakpoint, rerun same line",
+                                                                    "Multiple active breakpoints" };
+
+        public static String BreakModeToString(Options.BreakMode value)
```

Comment:
Now that we have an array of display names, we can use the enum value to index into it: `BreakModeOptions[(int)value]`.